src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.php

```diff
@@ -103,7 +103,7 @@
             ->tag('serializer.normalizer', ['priority' => -950])
 
         ->set('serializer.normalizer.problem', ProblemNormalizer::class)
-            ->args([param('kernel.debug')])
+            ->args([param('kernel.debug'), '$translator' => service('translator')->nullOnInvalid()])
```

Comment:
should we really rely on the named parameter here ? We don't include parameter names in our BC policy except for attribute constructors.

Reply:
we have to because of the way SerializerPass works:
https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/Serializer/DependencyInjection/SerializerPass.php#L79